.env.example

```
@@ -264,6 +264,11 @@ GITHUB_APP_WEBHOOK_URL=
 GITHUB_APP_SETUP_URL=
 # Fallback used when the value is not stored in Vault.
 GITHUB_APP_WEBHOOK_SECRET=
+# One-click website signup (hosted deployments only — leave off for self-hosted).
+# Requires the App setting "Request user authorization (OAuth) during installation"
+# and <backend>/github/app/signup/callback registered as a Callback URL.
+HOSTED_SIGNUP_ENABLED=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HOSTED_SIGNUP_ENABLED key should go before the NEXT_PUBLIC_GITHUB_APP_SLUG key

(UnorderedKey)